ftl/php/builder.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This package defines the interface for orchestrating image builds."""
+
+import hashlib
+import os
+import subprocess
+import tempfile
+import logging
+import datetime
+
+from containerregistry.client.v2_2 import append
+from containerregistry.transform.v2_2 import metadata
+
+from ftl.common import builder
+
+_PHP_NAMESPACE = 'php-composer-lock-cache'
+_COMPOSER_LOCK = 'composer.lock'
+_COMPOSER_JSON = 'composer.json'
+
+
+class PHP(builder.JustApp):
+    def __init__(self, ctx):
+        self.descriptor_files = [_COMPOSER_LOCK, _COMPOSER_JSON]
+        self.namespace = _PHP_NAMESPACE
```

Comment:
We shouldn't need this on self since we never change it.

Reply:
I think we need it because it's called in builder_runner [here](https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/ftl/common/builder_runner.py#L75).

Reply:
Ah sorry, missed that in the recent refactor.